ecosystem/sep-0039.md

```diff
@@ -0,0 +1,155 @@
+
+## Preamble
+
+```
+SEP: 0039
+Title: Interoperability Recommendations for NFTs
+Authors: Tomer Weller <tomer@stellar.org>, Frederic Rezeau <frederic.rezeau@gmail.com>
```

Comment:
More recently we've been using folks GitHub handles rather than emails, for privacy. Unless the authors prefer their emails be included here, could you change it to follows?
```suggestion
Authors: Tomer Weller <@tomerweller>, Frederic Rezeau <@FredericRezeau>
```